### DISCUSSION TITLE
why EventHub scaler does not scaled down pods

### DISCUSSION THREAD

**Discussion by leonids2005** [kedacore/keda#1721] [Q&A / Need Help]
5 pods created to process EventHub messages (5 partitions) but after processing all messages I still have 5 pods running.
Any suggestion how to troubleshoot it?

**Comment by tomkerkhove** [comment]
Did you check the KEDA logs?

**Reply by leonids2005** [reply]
Yes, I did. and it is sorted now. I had several problems. 1. It was not clear that EntityPath should be added to the EH connection string and 2. you need manually create a azure-webjobs-eventhub container. It could have saved me a lot of time if this was clearly stated in the documentation.

Going through closed and opened issues both items have been already mentioned.

**Reply by zroubalik** [reply]
@leonids2005 happy to hear that the problem is sorted. Could you please open a PR in our [docs](https://github.com/kedacore/keda-docs) repo and help us impro the EventHub scaler docs?

**Reply by ragi-vuedata** [reply]
@leonids2005  Can you please share a sample code/snippet regarding `azure-webjobs-eventhub` container?

**Reply by noiano** [reply]
I'm facing the same issues... can anybody share the solution? :)
